src/controllers/userProfileController.js

```diff
@@ -28,16 +28,25 @@ class UserProfileController {
     try {
       const rating = await authorRating(request);
       const foundUser = await User.findOne({
-        where: { id: userId }
+        where: { id: userId },
+        include: [
+          {
+            model: Article,
+            as: 'articles',
+            attributes: ['slug', 'title', 'description', 'body', 'imgUrl']
```

Comment:
Do we need to include the body of each article while returning the user profile?

Reply:
Hi @akhilome, I was thinking we could have the body there to jolt the interest of the user to read the article, then probably have ellipsis or 'read more' there at the end of it since it will be too long, or what do you think?

Reply:
Hmm. That would be nice.

I, however, brought this up due to the state our mockups for this feature:

![image](https://user-images.githubusercontent.com/15332525/50819563-ab991000-132a-11e9-94c2-1e8c0aeaa776.png)

Another thing, for piquing the reader's interest, we could use the article description which is already attached to the article object being returned.

What do you think?

Reply:
Hmm, Ok I get your point. Description might work as well, I'll consult with the rest of the team to find out what they think and give you feedback on it.

Reply:
Alright. 👍

Reply:
🎉